### ISSUE TITLE
Karabo Integration into MMODA

### ISSUE THREAD

**Issue by Lukas113** [i4Ds/Karabo-Pipeline#545]
After iterating with the MMODA team, we have evaluated the following use-case:

Demo from Sky to dirty-image on their backend [renkulab](https://renkulab.io/), with the according MMODA template for input and output parameters. Configurable must be the most important parameters like sky-sources, observation-time, phase-center, which telescope, field of view. To what extend they must be configurable is unclear atm (e.g. sky-sources). The following things need to be considered:
- Freedom of choice for configurable parameters?
- How can we ensure that these configurations don't have memory, resource and time-issues?
- MMODA interface of the defined parameters?
- What about reproducibility and maintenance? Different Karabo-versions (breaking API), different dependency-resolving (rebuilding of environment), how much do we have to care for CSCS object storage backwards-compatibility?

That's my understanding of how we proceed from our last meeting with MMODA on Feb22. @lmachadopolettivalle and @anawas were also part of the meeting. Feel free to add or change something.

**Comment by Lukas113** [comment]
@sfiruch I think we should create an i4ds (or Karabo?) group on renkulab for repository-management.

**Comment by sfiruch** [comment]
> @sfiruch I think we should create an i4ds (or Karabo?) group on renkulab for repository-management.

Yeah, that sounds reasonable. Can you create one for Karabo? Ideally the project wouldn't be I4DS/FHNW-specific. (Somewhat tangential: I also thought that we should move all Karabo repos into a Karabo GH org)

**Comment by Lukas113** [comment]
The initial setup is done on [karabo-dirty-image-sim](https://gitlab.renkulab.io/astronomy/mmoda/karabo-dirty-image-sim) repository. I wasn't able to a Karabo or i4ds group, because the MMODA-bot needs the repository to live in the `astronomy/mmoda` group. So you have to request access to the repository if you want to contribute. The most important information about how MMODA is working can be found in their [dev-guide](https://odahub.io/docs/guide-development/).
